### PR TITLE
Adds adopters up to jul 29th

### PR DESCRIPTION
**Pull request by bsnchan**
# Changes

- adds adopters up to july 29th

### CODE REVIEW

File: ADOPTERS.MD [hunk 1]
```diff
@@ -10,8 +10,12 @@ If you would like to be included in this table, please submit a comment to [this
...
+| Red Hat            | Red Hat OpenShift Serverless            | Generally Available | [link](https://www.openshift.com/learn/topics/serverless)            |
+| Scaleway           |                                         | Public Beta         | [link](https://www.scaleway.com/en)                                  |
 | Tata Communication |                                         | Development         | [link](https://www.tatacommunications.com/solutions/cloud/platforms) |
 | TriggerMesh        |                                         | Production          | [link](https://triggermesh.com)                                      |
 | VMware             | Cloud Native Runtimes for VMware Tanzu  | Generally Available | [link](https://network.tanzu.vmware.com/products/serverless)         |
 | VMware             | VMware Event Broker Appliance           | Development         | [link](https://vmweventbroker.io/)                                   |
+| deepc.ai           |                                         | Production          | [link](https://www.deepc.ai/)                                        |
```
Comment (vaikas) [inline ADOPTERS.MD:21]: How are we thinking about sorting this? Append, alphabetic, or something else? If Alphabetic (which makes sense to me, and looks like you did that), wouldn't this go before Fulcrum?

Comment (bsnchan) [reply ADOPTERS.MD:21]: I did :sort in vim and I think it put deepc.ai last because it's lowercase. Thoughts?

Comment (vaikas) [reply ADOPTERS.MD:21]: I'd rather have them alphabetic and not include capitalization in the sorting.

Comment (bsnchan) [reply ADOPTERS.MD]: updated on the latest commit :) PTAL

**Review by rhuss — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**